typedb/stream/bidirectional_stream.py

```diff
@@ -62,6 +62,9 @@ def stream(self, req: transaction_proto.Transaction.Req) -> Iterator[transaction
         self._dispatcher.dispatch(req)
         return ResponsePartIterator(request_id, self, self._dispatcher)
 
+    def done(self, request_id: UUID):
+        self._response_collector.remove(request_id)
+
```

Comment:
we now clean up completed `single`/`stream` from the response collectors, so we don't propagate and print errors for every old operation the transaction also handled but isn't active anymore!